Tests/UnitTests/Plugins/Geant4/Geant4DetectorSurfaceFactoryTests.cpp

```diff
@@ -144,20 +147,28 @@ BOOST_AUTO_TEST_CASE(Geant4DetecturSurfaceFactory_Transforms) {
 
   G4Transform3D nominal;
 
-  Acts::Geant4DetectorSurfaceFactory factory;
+  Acts::Geant4DetectorSurfaceFactory::Config factoryConfig;
+  Acts::Geant4DetectorSurfaceFactory factory(factoryConfig);
   factory.construct(cache, nominal, *worldPV, options);
 
   auto [element, surface] = cache.sensitiveSurfaces.front();
   BOOST_CHECK_EQUAL(surface->type(), Acts::Surface::SurfaceType::Plane);
 
   auto center = surface->center(gctx);
   auto normal = surface->normal(gctx, center, Acts::Vector3(1, 0, 0));
+
+  // 0.836516 -0.224144       0.5   45.9808
+  //  0.524519  0.591506 -0.612372   137.886
+  // -0.158494  0.774519  0.612372   144.957
+  //       0         0         0         1
```

Comment:
I am not sure if I understand the meaning of the values in this matrix. Maybe, we could add a line describing the content like `// Current values of the surface`?